sdk/vision/azure-ai-vision-imageanalysis/src/test/java/com/azure/ai/vision/imageanalysis/ImageAnalysisClientTest.java

```diff
@@ -0,0 +1,96 @@
+import java.util.AbstractMap.SimpleEntry;
+import java.net.MalformedURLException;
+
+import org.junit.jupiter.api.Test;
+import com.azure.ai.vision.imageanalysis.*;
+import com.azure.ai.vision.imageanalysis.models.*;
+
+class ImageAnalysisClientTest extends ImageAnalysisClientTestBase {
+
+    private final Boolean sync = true; // All tests in this file use the sync client
+
+    /***********************************************************************************
+     *
+     *                            HAPPY PATH TESTS
+     *
+     ***********************************************************************************/
+
+    @Test
+    public void testAnalyzeSyncAllFeaturesFromUrl() throws MalformedURLException {
+
+        createClientForStandardAnalysis(sync);
+
+        String methodName = new Object(){}.getClass().getEnclosingMethod().getName();
+        String imageSource = imageUrl;
+        List<VisualFeatures> visualFeatures =  Arrays.asList(
+                VisualFeatures.SMART_CROPS,
+                VisualFeatures.CAPTION,
+                VisualFeatures.DENSE_CAPTIONS,
+                VisualFeatures.OBJECTS,
+                VisualFeatures.PEOPLE,
+                VisualFeatures.READ,
+                VisualFeatures.TAGS);
+        Boolean genderNeutralCaption = true;
+        String language = "en";
+        List<Double> aspectRatios = Arrays.asList(0.9, 1.33);
+        String modelVersion = "latest";
+        ImageAnalysisOptions options = new ImageAnalysisOptionsBuilder()
+            .setLanguage(language)
+            .setGenderNeutralCaption(genderNeutralCaption)
+            .setSmartCropsAspectRatios(aspectRatios)
+            .setModelVersion(modelVersion)
+            .build();
+        doAnalysis(methodName, sync, imageSource, visualFeatures, options);
+    }
+
+    @Test
+    public void testAnalyzeSyncSingleFeatureFromFile() throws MalformedURLException {
+
+        // Note: The test is missing automatic validation that the connection URL includes "&key1=value1&key2=value2". Need
+        // to do this manually.
+        List<Entry<String, String>> queryParams = new ArrayList<>();
+        queryParams.add(new SimpleEntry<>("key1", "value1"));
+        queryParams.add(new SimpleEntry<>("key2", "value2"));
+        createClientForStandardAnalysis(sync, queryParams);
+
+        String methodName = new Object(){}.getClass().getEnclosingMethod().getName();
+        String imageSource = imageFile;
+        ImageAnalysisOptions options = null;
+
+        List<VisualFeatures> visualFeatures =  Arrays.asList(VisualFeatures.CAPTION);
+        doAnalysis(methodName + ":Caption", sync, imageSource, visualFeatures, options);
+
+        visualFeatures =  Arrays.asList(VisualFeatures.READ);
+        doAnalysis(methodName + ":Read", sync, imageSource, visualFeatures, options);
+
+        visualFeatures =  Arrays.asList(VisualFeatures.TAGS);
+        doAnalysis(methodName + ":Tags", sync, imageSource, visualFeatures, options);
```

Comment:
Same comment with multiple tests

Reply:
Thanks! see my comment in ImageAnalysisAsyncClientTest.java